embedx/config.schema.json

```diff
@@ -372,6 +372,14 @@
           "description": "The global maximum depth on all read operations. Note that this does not affect how deeply nested the tuples can be. This value can be decreased for a request by a value specified on the request, only if the request-specific value is greater than 1 and less than the global maximum depth.",
           "minimum": 1,
           "maximum": 65535
+        },
+        "max_read_width": {
+          "type": "integer",
+          "default": 100,
```

Comment:
Is this 100 queries?

Reply:
Yes basically. A single request could still have up to `max_read_width ^ max_read_depth` queries. The only way to significantly reduce these numbers is implementing caches.